app/Filament/Customer/Widgets/CustomerDashboardWidget.php

```
@@ -0,0 +1,96 @@
+        $totalPaye = (float) CustomerInvoice::where('client_id', $thirdParty->id)
+            ->whereIn('status', [InvoiceStatus::PAID, InvoiceStatus::PARTIALLY_PAID])
+            ->sum('total_ttc');
+
+        $totalPaye = (float) CustomerInvoice::where('client_id', $thirdParty->id)
+            ->whereIn('status', [InvoiceStatus::PAID, InvoiceStatus::PARTIALLY_PAID])
+            ->get()
+            ->sum('total_allocated');
+
+        $soldeDue = max(0, $totalFacture - $totalPaye);
+
+        $devisEnAttente = CustomerOrder::where('client_id', $thirdParty->id)
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Commerce\CustomerOrder::where().


(staticMethod.notFound)